tests/services/users/test_service_users.py

```diff
@@ -161,7 +181,9 @@ def test_search_permissions(app, db, user_service, user_moderator, user_res):
     """Test service search for permissions."""
     # User can search for himself
     search = user_service.search(
-        user_res.identity, q=f"username:{user_res._user.username}"
+        user_res.identity,
+        q=user_res._user.username,
+        fields=["username"],
```

Comment:
Can you explain why specifying the field `username`?

Reply:
We changed the query parser from query_string to multi_match(suggest API), the test before was using lucene syntax `username:`
Here it is changed because multi_match doesn't support lucene